tests/test_plugins/test_adjoint.py

```diff
@@ -1482,12 +1490,26 @@ def test_polyslab_structures(use_emulated_run):
         medium=td.Medium(),
     )
 
+    flux_mnt = td.FieldMonitor(
+        center=(0, 0, 0),
+        size=(1, 1, 0),
+        freqs=1e14 * np.array([1, 2, 3]),  # this previously errored
+        name="flux",
+    )
+
     VERTICES = np.array(
         [[-1.5, -0.5, -0.5], [-0.5, -0.5, -0.5], [-1.5, 0.5, -0.5], [-1.5, -0.5, 0.5]]
     )
     FACES = np.array([[1, 2, 3], [0, 3, 2], [0, 1, 3], [0, 2, 1]])
     STL_GEO = td.TriangleMesh.from_trimesh(trimesh.Trimesh(VERTICES, FACES))
 
+    from ..test_components.test_custom import CUSTOM_MEDIUM
```

Comment:
very nit but why not import at top level?

Reply:
could, I can change it.